maxim/logger/gemini/client.py

```
@@ -156,23 +186,51 @@ def send_message_stream(
     def __getattr__(self, name: str) -> Any:
-        result = getattr(self._chats, name)
-        return result
+        """Get an attribute from the chat session.
+
+        Args:
+            name: The name of the attribute.
+
+        Returns:
+            Any: The attribute.
+        """
+        return getattr(self._chat, name)        
```

Flake8 (7.2.0)

[warning] trailing whitespace

(W291)